deploy/helm/sumologic/templates/setup/setup-job.yaml

```diff
@@ -16,19 +16,30 @@ spec:
     spec:
       restartPolicy: OnFailure
       serviceAccountName: {{ template "sumologic.fullname" . }}-setup
+      volumes:
+      - name: setup
+        configMap:
+          name: {{ template "sumologic.fullname" . }}-setup
+          defaultMode: 0777
```

Comment:
Can be removed since kubernetes doesn't honor this field for configmap?

Reply:
This one needs to stay